plugins/optimization-detective/class-od-url-metric.php

```diff
@@ -417,6 +428,18 @@ public function get_uuid(): string {
 		return $this->data['uuid'];
 	}
 
+	/**
+	 * Gets ETag.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return non-empty-string|null ETag.
+	 */
+	public function get_etag(): ?string {
```

Comment:
We'll need to add a trivial test for this function.

Reply:
Added in b225fba.